README.md

```diff
@@ -122,6 +122,7 @@ and the Configuration API:
 - `REACT_APP_IMAGE_SIZE`:the maximum allowed image size of the Tenant icon
 - `REACT_APP_LOG_LEVEL`:the console level (trace,debug,info,warn,error,silent)
 - `REACT_APP_KEYCLOACK_ADMIN`: the endpoint to retrive realm data
+- `FIWARE_SERVICE`:the url of fiware
```

Comment:
FIWARE_SERVICE->ORION

the endpoint of orion context broker (this is just for demo purposes)